idaes/core/solvers/petsc.py

```diff
@@ -420,8 +420,8 @@ def petsc_dae_by_time_element(
     initial_variables=None,
     detect_initial=True,
     skip_initial=False,
-    snes_solver="petsc_snes",
-    snes_options=None,
+    initial_solver="petsc_snes",
```

Comment:
Seeing as we are pushing to get backward compatibility tests into place, this look like a place where we are going to need backward compatibility options and warnings. I.e. we need to keep the old options around, and have them raise deprecation warnings if they are used before sending the value to the new argument (and raise an exception if both old and new are provided).

Reply:
If it is that new then we are probably OK, but we need to double check that is the case.